packages/syftcli/syftcli/bundle/create.py

```diff
@@ -31,43 +30,56 @@ class ContainerEngineType(str, Enum):
 
 
 def create(
-    version: str = "latest",
+    version: Annotated[str, Option("--version", "-v")] = "latest",
     outdir: Annotated[
-        Path, Option(dir_okay=True, file_okay=False, writable=True)
+        Path, Option("--outdir", "-d", dir_okay=True, file_okay=False, writable=True)
     ] = DEFAULT_OUTPUT_DIR,
-    engine: ContainerEngineType = ContainerEngineType.Docker,
+    engine: Annotated[
+        ContainerEngineType, Option("--engine", "-e")
+    ] = ContainerEngineType.Docker,
     dryrun: bool = False,
 ) -> None:
-    """Create an offline container image bundle for a given Syft version."""
+    """Create an offline deployment bundle for Syft."""
 
     # Validate Syft version. Fails if version is not valid.
     ver = validate_version(version)
 
-    # Prepare output directory
-    outdir = outdir.expanduser().resolve()
-    outdir.mkdir(parents=True, exist_ok=True)
-    tarpath = Path(outdir, f"syft-{ver.release_tag}-{engine}.tar")
+    # Prepare temp paths
+    out_path = prepare_output_dir(outdir)
+    temp_path = prepare_tmp_dir(out_path)
+    img_path = Path(temp_path, "images.tar")
+
+    # prepare output paths
+    bundle_path = Path(out_path, f"syft-{ver.release_tag}-{engine}.tar")
```

Comment:
![Screenshot 2023-08-31 at 5 35 55 PM](https://github.com/OpenMined/PySyft/assets/43314053/b1697027-39dd-4dbb-bbd2-41095ceae2de)
MAYBE: While we create the container engine, it is of type, ContainerEngine.Docker, we could make it plain `docker`

Reply:
weird - works perfectly fine on my setup. regardless I'll push a fix